scss/widgets/generic/tabPanel/_index.scss

```diff
@@ -123,13 +145,17 @@ $generic-tabpanel-inset-shadow-bottom: inset 0 (-$generic-tabs-border-width);
     }
   }
 
-  @include tabpanel-shadow-mixin($tabs-border-color);
+  @include tabpanel-shadow-mixin($tabs-border-color, 'dx-tab-selected');
 
   &.dx-state-focused {
-    @include tabpanel-shadow-mixin($tabs-focused-border-color);
+    @include tabpanel-shadow-mixin($tabs-focused-border-color, 'dx-state-focused');
   }
 
   .dx-multiview-wrapper {
     border: $generic-tabs-border;
+
+    &.dx-state-focused {
+      border-color: $tabs-focused-border-color;
+    }
```

Comment:
@EugeniyKiyashko  please review scss

Reply:
PR in progress, scss is going to change.

Reply:
@EugeniyKiyashko, could you continue the review?

Reply:
I'll trust you